client/dvm/consensus/Cargo.toml

```diff
@@ -4,20 +4,21 @@ description = "Consensus for darwinia dvm"
 edition     = "2018"
 homepage    = "https://darwinia.network/"
 license     = "GPL-3.0"
-name        = "dvm-consensus"
+name        = "dc_consensus"
```

Comment:
Most crates are using slash `-`, it is weird to use underscore `_` here.

Reply:
Recommend to use `-`